chia/rpc/rpc_server.py

```diff
@@ -324,13 +325,15 @@ async def start_rpc_server(
         daemon_connection = asyncio.create_task(rpc_server.connect_to_daemon(self_hostname, daemon_port))
     runner = web.AppRunner(app, access_log=None)
     await runner.setup()
+
```

Comment:
I was going to suggest that asking for the port later (like `get_port()`) might be a nicer model. But I realize we drop the task handle on the floor here and just detach it here.. so I suppose that isn't an option until this is fixed

Reply:
We can't just directly start using this, but maybe look at https://trio.readthedocs.io/en/stable/reference-core.html#trio.Nursery.start would be useful as a reference for launching concurrent tasks and getting some info out of them after they 'start up'.